ext/dapr-ext-workflow/dapr/ext/workflow/workflow_runtime.py

```diff
@@ -31,10 +32,15 @@ class WorkflowRuntime:
     """WorkflowRuntime is the entry point for registering workflows and activities.
     """
 
-    def __init__(self, host: Optional[str] = None, port: Optional[str] = None):
+    def __init__(self, host: Optional[str] = None, port: Optional[str] = None, apiToken: Optional[str] = None):
+        metadata = []
+        if apiToken is not None:
```

Comment:
Same feedback [not blocking] - do you really need this in the method signature?

In Python we really want folks to use our settings package to centrally set these things (if they are not already provided via environment variables)

Reply:
yeah, removed the option